src/lib/formatter/RuleResultRecombinator.ts

```diff
@@ -344,10 +344,24 @@ URL: ${url}`;
 		return {columns, rows};
 	}
 
-	private static constructJson(results: RuleResult[]): string {
+	private static constructJson(results: RuleResult[], verboseViolations = false): string {
 		if (results.length === 0) {
 			return '';
 		}
+
+		if (verboseViolations) {
+			for (const result of results) {
+				if (result.engine === 'retire-js') {
```

Comment:
Don't use `retire-js` as a hardcoded string.
Add this import:
`import {ENGINE} from '../../Constants';
and then change this line to
`if (result.engine === ENGINE.RETIRE_JS) {`